modules/articles/tests/server/article.server.routes.tests.js

```diff
@@ -306,7 +306,10 @@ describe('Article CRUD tests', function () {
     var articleObj = new Article(article);
 
     // Save the article
-    articleObj.save(function () {
+    articleObj.save(function (err) {
+      if (err) {
```

Comment:
@jrodenbostel If the intention is to catch/handle unexpected (possibly random) errors here, then why not just fail the test when `err` exists? This test should only pass when all conditions and expectations are met, right?

Maybe I'm thinking about it wrong. I guess we have two options, either we assert that `err` doesn't exist to fail the test when it does exist, or we pass the err to the callback to propagate the actual error back to the test runner; perhaps, the former solves both?